extensions/reactive-datasource/deployment/src/main/java/io/quarkus/reactive/datasource/deployment/VertxPoolBuildItem.java

```diff
@@ -12,26 +12,23 @@
  */
 public final class VertxPoolBuildItem extends MultiBuildItem {
```

Comment:
Should we mark it as deprecated and `forRemoval` then?

Reply:
Good point!

Reply:
Done